app/src/main/java/com/zulip/android/activities/RecyclerMessageAdapter.java

```diff
@@ -381,10 +384,22 @@ public void onBindViewHolder(final RecyclerView.ViewHolder holder, final int pos
                             .setOnClickListener(new View.OnClickListener() {
                                 @Override
                                 public void onClick(View view) {
-                                    Intent i = new Intent(Intent.ACTION_VIEW);
-                                    i.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
-                                    i.setData(Uri.parse(url));
-                                    zulipApp.startActivity(i);
+                                    if (Build.VERSION.SDK_INT < 15)
+                                    {
+                                        Intent i = new Intent(Intent.ACTION_VIEW);
+                                        i.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+                                        i.setData(Uri.parse(url));
+                                        zulipApp.startActivity(i);
+                                    }
+                                    else
+                                    {
+                                        CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder();
+                                        builder.setStartAnimations(zulipApp.getZulipActivity(), R.anim.slide_in_right, R.anim.slide_out_left);
```

Comment:
@kunall17 I tried this but it was not working in chrome custom tabs when i click the the exit button.